_content/articles/mortality-of-the-dalai-lama_maccormack-ian.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Mortality of the Dalai Lama and its Scriptural Sources: More on Tibetan Political Theology"
+authors:
+  - "Ian MacCormack"
+external_url: "https://poj.peeters-leuven.be/content.php?url=article&id=3291580&journal_code=JIABS"
+drive_links:
+  - "https://drive.google.com/file/d/1x0mAq8NUVrF28cRu6Bmqrl2K0TtXJaQq/view?usp=drive_link"
+course: tibetan
+tags:
+  - tibet
+  - political-ideology
+  - society
+year: 2022
+journal: jiabs
+volume: 45
+pages: "205--251"
+---
+
+> The prospect of embodied divinity was not uncommon in Tibetan discourse and practice in general. The point is that this choice of terms forces the issue. To assert a human god’s ongoing karmic dependency is
+to define divinity and mortality in terms that are direct inversions of one another.
+
+In this final part of a two-part study the 5th Dalai Lama's divine kingship, the author explores how the concept of residual karma explains the Dalai Lama’s mortality despite his spiritual perfection. Drawing on scriptural sources and the writings of the Desi—a secular regent in Tibetan governance—the essay shows how this concept helped reconcile divine kingship with the human realities of suffering and death.
+
+Part one of this study can be found [here](/content/articles/divinity-of the-dalai-lama-scriptural-sources_maccormack-ian.md)
```

Comment:
`/content/articles/divinity-of-the-dalai-lama-scriptural-sources_maccormack-ian` No `.md` in the link URLs 😊